src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -100,15 +100,18 @@ def get_entry_output_types(output_types):
 
 
 def get_filename(file_type, file_prefix, matrix_combo):
-    prefix = ""
+    file_type_prefix = ""
     file_ext = ""
     if file_type == str(OutputTypes.CONDA):
         file_ext = ".yaml"
     if file_type == str(OutputTypes.REQUIREMENTS):
         file_ext = ".txt"
-        prefix = "requirements_"
+        file_type_prefix = "requirements_"
```

Comment:
Couple this with the edit below.
```suggestion
        file_type_prefix = "requirements"
```